doc/user/data/sql_funcs.yml

```
@@ -512,6 +512,10 @@
     description: The `uuid` uniquely identifying this Materialize environment.
     unmaterializable: true
 
+  - signature: 'uuid_generate_v5(namespace: uuid, name: text) -> uuid'
+    description: 'Generates a version 5 UUID (using SHA-1) in the given namespace using
+      the specified input name. [See the postgres docs for more details](https://www.postgresql.org/docs/current/uuid-ossp.html)'
```

We don't generally link to the PostgreSQL docs in these function descriptions and this doesn't strike me as underdescribed enough to warrant it! (It's kind of a bad look if our documentation isn't able to stand on its on; we do it in a _few_ key places but would rather not make a habit of it.) Also those `uuid_ns_` functions that the PostgreSQL doc talks about just aren't available, so I think it's likely to be as frustrating as it is helpful.

```suggestion
      the specified input name.
```

---

Side note, since I missed the boat on #17834: `mz_environment_id()` should move under `type: System information`.